README.md

```diff
@@ -1,2 +1,83 @@
 # in-toto
 in-toto is a series of scripts to protect the supply chain integrity.
+
+# About in-toto
+in-toto guarantees that the end-user (or client) is able to verify that the entire development life cycle has been conducted as per the specified layout and that each of the functionaries (eg. developers) have performed the specified tasks and there haven't been any malicious changes in the files.
+
+in-toto requires a project layout that specifies the functionaries and the tasks they are supposed to perform.
+After each functionary performs its task, a link metadata is generated.
+This metadata is used to verify the intermediate and final products with the project layout.
```

Comment:
This feels a little bit ambiguous. What about:

> This metadata file is used to verify that the materials used and the products produced within this task weren't tampered with. The link metadata also contains a signature to ensure that the right functionary performed this task.

Or something along those lines